midi_app_controller/actions/napari_actions.py

```diff
@@ -67,6 +68,27 @@ def zoom_in(viewer: Viewer) -> None:
     viewer.camera.zoom /= 0.95
 
 
+def _replace_value_at(seq, pos, new_val):
+    tmp_seq = list(seq)
+    tmp_seq[pos] = new_val
+    return tuple(tmp_seq)
+
+
+def increment_axis(viewer: Viewer, slider_num=None, inc=1) -> None:
+    not_disp = viewer.dims.not_displayed
+    if slider_num is None:
+        axis = viewer.dims.last_used
+    elif slider_num < len(not_disp):
+        axis = not_disp[slider_num]
+    else:
+        return
+    cur = viewer.dims.current_step
+    next_value = viewer.dims.current_step[axis] + inc
```

Comment:
Did you forget about modulo operation after reaching maximum position?

Reply:
See comment immediately under this line